models/User.js

```diff
@@ -17,6 +17,10 @@ const UserSchema = new Schema(
       type: String,
       required: true,
     },
+    games: {
+      type: Array,
```

Comment:
`games: [{ type: ObjectId, required: true }]`

Would it be better to say that the games are an array of objectIds?

Reply:
As it is with that, it's working as intended with the stats page and games is already an array